kitchen.ec2.yaml

```diff
@@ -0,0 +1,68 @@
+---
+driver:
+  name: ec2
+  aws_ssh_key_id: <YOUR_KEY_NAME>
+  security_group_ids: [ "<YOUR_SECURITY_GROUP_ID>" ]
+  region: <YOUR_REGION>
+  availability_zone: a
+  subnet_id: <YOUR_SUBNET_ID>
+  #iam_profile_name: chef-client
+  instance_type: t2.micro
+  associate_public_ip: true
+  interface: public
+  transport:
+    username: ec2-user
```

Comment:
This should be removable: it is defined below.

Reply:
Done